step03.md

```diff
@@ -84,7 +84,9 @@ $ node server.js
 
 And if you are using cloud9:
 
-Simply click the `Run` button on the top menu.
+- Select the file `server.js` in the workspace folder tree.
+- Right click the `Run` button on the top menu and select `New Run Configuration` from the menu. This will open the Run Panel at the bottom of your workspace.
+- Type `node server.js` in the Command text editor and press Enter.
```

Comment:
@lauragieco , im not sure what im doing differently. But I get this error on entering node server.js
```
Error: Cannot find module '/home/ubuntu/workspace/node'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.runMain [as _onTimeout] (module.js:441:10)
    at Timer.listOnTimeout (timers.js:92:15)
```
I dont get this error if I just click Run (no editing of the command) after selecting the file.
Also dont get this error if I enter node server.js from the terminal.

Why not just keep things simple and just leave it at: 
> click the `Run` button on the top menu